.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+*                            @hosseinsh, @csine-nflx, @charhate, @jtschladen
```

Comment:
Should this use spaces rather than commas? [Doc](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) is unclear but the one example with multiple usernames has spaces

Reply:
you are right, I see spaces in the examples